docs/integrations.md

```diff
@@ -40,6 +40,46 @@ The Go template can utilize helper functions from [Sprig].
 
 For example templates, see [tracee/cmd/tracee-rules/templates].
 
+## Container Enrichment
+
+Tracee is capable of extracting information on running containers on your system during runtime through tracking created cgroups in kernel and can further enrich the container events from data queried by communicating with the relevant container's runtime and SDK from userspace.
+
+Since this feature is still experimental, it is disabled by default and must be explicitely enabled.
+
+In order to enable this capability in the container's trace mode or with tracee-ebpf's binary, use the `--containers` flag.
+If running tracee with tracee-rules, set the environment flag `CONTAINERS_ENRICHMENT` (example below).
+
+If running tracer-ebpf directly from binary, it will automatically search for known supported runtimes in their default socket's locations.
+
+However, when running tracee from a container, the runtime sockets must be manually mounted in order for the enrichment features to work.
+
+Using containerd as our runtime for example, this can be done by running tracee like so:
+```shell
+   docker run \
+     --name tracee --rm -it \
+     --pid=host --cgroupns=host --privileged \
+     -v /etc/os-release:/etc/os-release-host:ro \
+     -v /var/run/containerd:/var/run/containerd
+     -e LIBBPFGO_OSRELEASE_FILE=/etc/os-release-host \
+     -e CONTAINERS_ENRICHMENT=1 \
+     aquasec/tracee:latest
+```
+
+Most container runtimes have their sockets installed by default in `/var/run` so if your system includes multiple container runtimes, tracee can track them all, however one should then mount either all their runtime sockets or `/var/run` in it's entirety.
+
+Currently, tracee will look in the following paths for autodiscovering the listed runtimes:
+
+1. Docker:     `/var/run/docker.sock`
+
+2. Containerd: `/var/run/containerd/containerd.sock`
+
+3. CRI-O:      `/var/run/crio/crio.sock`
+
+!!! note
```

Comment:
Same as above, maybe bold or `###` instead

Reply:
That `!!! note` renders it inside a box, at least how it renders for me.

Reply:
ditto